### DISCUSSION TITLE
[GPU] Quick question on GPU used by Panda3D

### DISCUSSION THREAD

**Discussion by mwulfman** [Wandercraft/jiminy#830] [Q&A] [answered]
Hello,

Does anyone know if there is a way to specify which GPU to be used for rendering the simulation (using Panda3D in my case) when using a multi-GPU computer ? And is there any limitation if rendering with a AMD GPU ?

Thank you for your help !

**Comment by duburcqa** [comment, marked as answer]
There is several approaches. You can specified the GPU available at process-level using Nvidia-specific environment variables. Another option is the use the panda3D-specific configuration option `egl-device-index 0`, but I'm not sure there is a way at the moment to pass this kind of custom options to panda3d instances managed at jiminy-level.

For reference: https://github.com/panda3d/panda3d/issues/1306, https://stackoverflow.com/questions/39649102/how-do-i-select-which-gpu-to-run-a-job-on
